content/community/nominating-issues/_index.en.adoc

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
What is the need for this intermediate page? Will we be having more sub-pages here in the future?

Reply:
I think the logic here was that it doesn't really match the "Contributing Code" header. We do already have "Raising Issues" there though so I'm fine with moving it (Also worth noting that page is quite out of date).

Reply:
I think maybe it does belong in a separate section, it just doesn't need the additional level, so it could be at the same level as "Contributing code" and "Code of Conduct" etc, but without any sub-page underneath it.
We can always make a section later if more pages go into it.

Maybe the phrase "Nominating an Issue" isn't very clear (I didn't get what it was until I read the page). I suggest "Nominating issues in sprint planning" or similar